### PR TITLE
Weekly

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Describe your changes. This will most likely be a copy-paste of your commit
  message(s), which should be descriptive and informative.

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-125465](https://oktainc.atlassian.net/browse/OKTA-125465)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@rchild-okta or @johngronberg-okta or @jmelberg-okta 
first merge of 'weekly' into master for the new process.

### CODE REVIEW

File: _source/_docs/api/resources/oauth2.md [hunk 1]
```diff
@@ -532,6 +532,44 @@ invalid_grant           | The *code* or *refresh_token* value was invalid, or th
 unsupported_grant_type  | The grant_type was not *authorization_code* or *refresh_token*. |
 invalid_scope           | The scopes list contains an invalid or unsupported value.    |
 
+#### Request Example: Resource Owner Password Flow
+
+~~~
+curl -X POST \
```
Comment (jmelberg-okta) [inline _source/_docs/api/resources/oauth2.md]: These should be double quotes to keep with the same format as previous CURL blocks

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oauth2.md]: Ack. I'll do this in a follow-up, this is the merge from weekly to master.

**Comment by jmelberg-okta** [comment]
@mystiberry-okta Turns out we have a few broken tables, we should fix these and add tests as a follow up commit.

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 2]
```diff
@@ -107,21 +111,9 @@ As noted above, to change user state, the `self` link should be called to retrie
...
-For more information, see the [Identity Providers API](/docs/api/resources/idps.html#social-authentication-token-operation). <!-- OKTA-118687 -->
-
-### Platform Bugs Fixed
+### Platform Bug Fixed
 
- * Searches on [User](http://developer.okta.com/docs/api/resources/users.html#list-users-with-search) incorrectly returned deleted users or out-of-date user status in some cases. (OKTA-116928)
- * Some orgs were unable to add OpenID Connect or OAuth 2.0 clients to an access policy in a custom Authorization Server. (OKTA-117630)
- * When deleting a claim from a custom Authorization Server, the Delete dialog didn't close after clicking **OK** or **Cancel**. (OKTA-124271)
- * Read-only Administrator user interface didn't exactly match that role's access rights. (OKTA-123116)
+Some queries on `/api/v1/apps` with incorrect filter parameters returned an empty `_embedded node` in the response instead of the correct error message. (OKTA-124544)
```
Comment (jmelberg-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: Why is this included in `(parenthesis)`, while others are in a `<!-- comment -->`?

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: The comments are hidden, the parens are printed so people can see them. We only print the bug numbers, but writers can't keep track of everything if they don't include the numbers somewhere for the new features/enhanced features...

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: ack on the broken tables as well. Thank you!
